app/views/shared/summary_cards/_resource.html.slim

```diff
@@ -23,13 +23,11 @@
       .clearfix
     .summary-card__row
       p
-        - if resource.url?
-          = link_to resource.url, resource.url, target: '_blank'
```

Comment:
Out of curiosity, why were the URLs being shown in this section instead of the description? Also, removing this line won't remove the "View Original" link, correct?